packages/block-editor/src/components/alignment-toolbar/index.js

```diff
@@ -54,6 +54,7 @@ export function AlignmentToolbar( props ) {
 				return {
 					...control,
 					isActive,
+					role: 'menuitemradio',
```

Comment:
This works great. I wonder if we could have a `Toolbar` props called `type`, which could be either `radio`, `checkbox`, or nothing, and pass that down to `DropdownMenu` to handle the roles etc.

Reply:
And when a `Toolbar` is not collapsed, the `type` wouldn't do anything.

Reply:
@diegohaz is working on some improvements around that in https://github.com/WordPress/gutenberg/pull/18931. One thing we discussed is to have `ToolbarGroup` which can be collapsible, and one way would be to have this type `radio` or `checkbox` as proposed. We might also try a more declarative way with JSX rather than using `controls` props. It needs to be further investigated.

Aside: I probably used my limit of pings for Diego for this year :)